manage.js

```diff
@@ -0,0 +1,127 @@
+const express = require('express');
+const app = express();
+const la = require('./luis-add');
+const lr =require('./luis-read');
+const ld =require('./luis-delete');
+const lu =require('./luis-update');
+const da = require('./dialogflow-add');
+const dr =require('./dialogflow-read');
+const dd =require('./dialogflow-delete');
+const du =require('./dialogflow-update');
+const bodyParser = require('body-parser');
+const urlencodedParser = bodyParser.urlencoded({ extended: false });
+app.use(express.static('public'));
+app.use(bodyParser.json());
+app.post('/add', urlencodedParser, function (req, res) {
+    if(req.body.destination==="Microsoft") {
+        la.add(JSON.parse(req.body.utterance))
+            .then((responses) => {
+                console.log(responses);
+            })
+            .catch(err=>{
+                console.log('ERROR:', err);
+            })
+    }else if(req.body.destination==="Google") {
+        //var projectId ="roomrservation";
+        da.add(JSON.parse(req.body.utterance),req.body.projectId)
+            .then((responses) => {
+                console.log(responses);
+            })
+            .catch(err => {
+                console.log('ERROR:', err);
+            })
+    }else{
+        res.end("Please enter the correct destination.")
+    }
+});
+
+app.post('/read', urlencodedParser, function (req, res) {
+    if(req.body.destination==="Google"){
+        //var intentId ='projects/roomrservation/agent/intents/791f16e2-d0d5-4012-9d9c-34307cf07cfe';
+        dr.read(req.body.intentId)
+            .then(responses =>{
+             return dr.tran(responses);
```

Comment:
Actually the `tran` is not an async function, so you don't need to put it into a `then` closure independently, you can just write your logic in one `then`.